plsync/mlabconfig_test.py

```diff
@@ -25,6 +84,8 @@ def setUp(self):
                                      self.users,
                                      nodegroup='MeasurementLabCentos')]
         self.attrs = [model.Attr('MeasurementLabCentos', disk_max='60000000')]
+        # Turn off logging output during testing (unless critical).
+        logging.disable(logging.CRITICAL)
```

Comment:
You're correct. Fixed. The docs say: "disable all logging calls of severity lvl and below" https://docs.python.org/2/library/logging.html#logging.disable